tsconfig.json

```
@@ -3,13 +3,14 @@
     "target": "esnext",
     "module": "esnext",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "jsx": "preserve",
+    "jsx": "react",
```

```suggestion
    "jsx": "preserve",
```

ref. https://github.com/agnosticful/chipstackoverflow/pull/7#issuecomment-594218904